Sources/RichTextKit/Context/RichTextContext.swift

```diff
@@ -55,114 +55,91 @@ public class RichTextContext: ObservableObject {
     public internal(set) var selectedRange = NSRange()
 
 
-    // MARK: - Observable properies
+    // MARK: - Public properies
 
+    /// Whether or not the text is currently being edited.
+    @Published
+    public var isEditingText = false
+    
+    /// The current text alignment, if any.
+    @Published
+    public var textAlignment: RichTextAlignment = .left
+    
+    /// The current font name.
+    @Published
+    public var fontName = ""
+    
+    /// The current font size.
+    @Published
+    public var fontSize = CGFloat.standardRichTextFontSize
+    
+    /// The currently highlighted range, if any.
+    public var highlightedRange: NSRange?
+    
+    /// Use this Publisher to emit any attribute changes to textView.
+    public let userActionPublisher: PassthroughSubject<RichTextAction, Never> = .init()
+    
+    // MARK: - Internal properties
+    
     /// The current background color, if any.
     @Published
-    public var backgroundColor: ColorRepresentable?
+    var backgroundColor: ColorRepresentable?
 
     /// Whether or not the current rich text can be copied.
     @Published
-    public var canCopy = false
+    var canCopy = false
```

Comment:
Why do you remove public on these?

Reply:
> Properties on Context are now internal, meaning if you want to set some attributes from the outside, use userActionPublisher! 

The idea is that those values inside context are used as flags for internal RichText to communicate to coordinator. I am actually not quite sure if this should be public API or not, because all the stuff you can actually get in real time from `RichTextComponent` and making public on both places seems redundant... But this is up to discussion as I can see some room for improvement in here.

Reply:
Yeah, I think with these amazing action extensions, there are probably a lot of things that can be removed.